kimchi/src/tests/foreign_field_add.rs

```diff
@@ -975,36 +996,55 @@ fn prove_and_verify_6() {
     prove_and_verify(6);
 }
 
-/*
+// Test with FFAdd gates without range checks
```

Comment:
Similar to Matthew's PR review request for ffMul, I think we should test ffAdd on both of curves: `Pallas` and `Vesta`.

This can now be done using the `TestFramework`, so I'd just update those tests that you have using `TestFamework`, rather than all of them to run on both curves.  You can see the ffMul gate to see how it's done.

Reply:
Done!

Reply:
Great! 👍🏼